tests/node-tests.rs

```diff
@@ -0,0 +1,366 @@
+use ft_io::*;
+use fungible_token::WASM_BINARY_OPT;
+use gclient::{EventListener, EventProcessor, GearApi, Result};
+use gstd::Encode;
+
+const USERS: &[u64] = &[3, 4, 5];
+
+#[tokio::test]
+#[ignore]
+async fn mint_test() -> Result<()> {
+    let api = GearApi::dev().await?;
+
+    let mut listener = api.subscribe().await?; // Subscribing for events.
+
+    // Checking that blocks still running.
+    assert!(listener.blocks_running().await?);
+
+    // Init
+    let init_ft = Initialize::Config(InitConfig {
+        name: String::from("MyToken"),
+        symbol: String::from("MTK"),
+        decimals: 18,
+    })
+    .encode();
+
+    let gas_info = api
+        .calculate_upload_gas(None, WASM_BINARY_OPT.to_vec(), init_ft.clone(), 0, true)
+        .await?;
+
+    let (message_id, program_id, _hash) = api
+        .upload_program_bytes(
+            WASM_BINARY_OPT.to_vec(),
+            gclient::now_micros().to_le_bytes(),
+            init_ft,
+            gas_info.min_limit,
+            0,
+        )
+        .await?;
+
+    assert!(listener.message_processed(message_id).await?.succeed());
+
+    // Mint
+    let mint = FTAction::Mint(1_000_000);
+    let gas_info = api
+        .calculate_handle_gas(None, program_id, mint.encode(), 0, true)
+        .await?;
+
+    let (message_id, _) = api
+        .send_message(program_id, mint, gas_info.min_limit, 0)
+        .await?;
+
+    assert!(listener.message_processed(message_id).await?.succeed());
+
+    assert!(listener.blocks_running().await?);
+
+    // Approve
+    let approve = FTAction::Approve {
+        to: USERS[1].into(),
+        amount: 500,
+    };
+
+    let gas_info = api
+        .calculate_handle_gas(None, program_id, approve.encode(), 0, true)
+        .await?;
+
+    let (message_id, _) = api
+        .send_message(program_id, approve, gas_info.min_limit, 0)
+        .await?;
+
+    assert!(listener.message_processed(message_id).await?.succeed());
+
+    assert!(listener.blocks_running().await?);
+
+    Ok(())
+}
+
+#[tokio::test]
+#[ignore]
+async fn migration_by_init() -> Result<()> {
+    let api = GearApi::dev().await?;
+
+    let mut listener = api.subscribe().await?; // Subscribing for events.
+
+    // Checking that blocks still running.
+    assert!(listener.blocks_running().await?);
+
+    migration_by_init_with_len(&api, &mut listener, 1, 1).await?;
+    migration_by_init_with_len(&api, &mut listener, 10, 1).await?;
+    migration_by_init_with_len(&api, &mut listener, 100, 1).await?;
+
+    for n in 1..=18u64 {
+        migration_by_init_with_len(&api, &mut listener, n * 1_000, 1).await?;
+    }
+    // No passed because 'Transaction would exhaust the block limits'
+    // migration_by_init_with_len(&api, &mut listener, 100_000, 1).await?;
```

Comment:
If this returns an error, why did you commented it out instead of `assert!`ing the error?